src/logdata/include/linetypes.h

```diff
@@ -44,7 +44,7 @@ using LinesCount = fluent::NamedType<uint64_t, struct lines_count, fluent::Addab
                                      fluent::Decrementable, fluent::Comparable, fluent::Printable>;
 
 using LineLength
-    = fluent::NamedType<int, struct line_length, fluent::Comparable, fluent::Printable>;
+    = fluent::NamedType<qsizetype, struct line_length, fluent::Comparable, fluent::Printable>;
```

Comment:
That changes the limits for maximum supported line length. `int` is used here because lines are sometimes stored in `QString` and in Qt5 does not support strings of 64-bit sizes.  Is using qsizetype everywhere realy required for Qt6 migration? What kind of errors/warning are produced without it?